schema/validator.js

```diff
@@ -0,0 +1,12 @@
+var Ajv = require('ajv');
+var ajv = new Ajv({allErrors: true});
+var json = require('./schema.json');
+
+module.exports = function validate(data) {
+	var validSchema = ajv.compile(json);
+	var valid = validSchema(data);
+
+	if(!valid) {
+		throw new Error("Your ExtractTextPlugin config is not correct. Please double check.");
```

Comment:
You can get possible error text through `ajv.errorsText()`. Better throw that instead.